packages/server/lib/plugins/child/ts_node.js

```diff
@@ -40,12 +40,24 @@ const getTsNodeOptions = (tsPath, registeredFile) => {
     }
   }
 
+  const escapeRegExp = (string) => {
```

Comment:
I'm not going to review this area until the `ts-node` issues are figured out 😅

Reply:
good call. I've fallen back to a lame but surgical fix. I'll fill people in at scrum.